packages/trader/src/AppV2/Containers/ContractDetails/contract-details.scss

```diff
@@ -11,7 +11,7 @@ $TWO_BUTTONS_FOOTER_HEIGHT: 120px;
     overflow-y: auto;
     overflow-x: hidden;
     gap: var(--semantic-spacing-general-sm);
-    height: 100%;
+    height: calc(100vh - 4rem);
```

Comment:
Is 4 rem connected with some quill-token based elements? If yes, we can use var(--core-size-2000) instead of 4 rem.
Please, ignore if irrelevant